pkg/kubectl/cmd/logs.go

```diff
@@ -195,7 +196,7 @@ func (o *LogsOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []str
 	switch len(args) {
 	case 0:
 		if len(o.Selector) == 0 {
-			return cmdutil.UsageErrorf(cmd, "%s", logsUsageStr)
+			return cmdutil.UsageErrorf(cmd, "%s", logsUsageErrStr)
```

Comment:
Don't, just print the error you think is valuable here, eg. `selector is required`, which btw. can be done with `cmd.MarkFlagRequired` function.